src/btree/bt_misc.c

```diff
@@ -8,6 +8,29 @@
 
 #include "wt_internal.h"
 
+/*
+ * __wt_addr_string --
+ *     Load a buffer with a printable, nul-terminated representation of an address.
+ */
+const char *
+__wt_addr_string(WT_SESSION_IMPL *session, const uint8_t *addr, size_t addr_size, WT_ITEM *buf)
+{
+    WT_BM *bm;
+    WT_BTREE *btree;
+
+    btree = S2BT_SAFE(session);
+
+    if (addr == NULL) {
+        buf->data = "[NoAddr]";
+        buf->size = strlen("[NoAddr]");
+    } else if (btree == NULL || (bm = btree->bm) == NULL ||
+      bm->addr_string(bm, session, buf, addr, addr_size) != 0) {
+        buf->data = "[Error]";
+        buf->size = strlen("[Error]");
```

Comment:
Since this is being used multiple times (including below), I suggest making it a macro, as well as the `[NoAddr]` above.